lib/src/util/yust_file_handler.dart

```diff
@@ -157,7 +157,7 @@ class YustFileHandler {
     if (yustFile.name == null || yustFile.storageFolderPath == null) {
       throw ('The file needs a name and a storageFolderPath to perform an upload!');
     }
-
+    yustFile.modifiedAt = Yust.helpers.utcNow();
```

Comment:
In the yust_file_picker the new file is created (l. 300-307). The argument **modifiedAt** should be assigned there.